test/contracts/acl/acl_params.js

```diff
@@ -114,7 +114,7 @@ contract('ACL params', ([permissionsRoot, specificEntity, noPermission, mockAppA
 
     // Both the assert and oog gas cases should be similar, since assert should eat all the
     // available gas
-    for (let [description, FailingOutOfGasOracle] of [
+    for (const [description, FailingOutOfGasOracle] of [
       ['asserts', AssertOracle],
       ['uses all available gas', OverGasLimitOracle],
     ]) {
```

Comment:
Similar to above, we might as well add a block to test the `OnlyOwnerOracle` :).

Reply:
Added! Let me know!